hack/e2e-test-ocp.sh

```diff
@@ -129,7 +129,7 @@ label_worker_nodes
 # supported in OCP. Also, skip Alpha features like TAS.
 # TODO: Remove Fair Sharing from the skip list once the issue is fixed.
 $GINKGO $GINKGO_ARGS \
-  --skip="(AppWrapper|JobSet|LeaderWorkerSet|Pod|Deployment|StatefulSet|Metrics|Fair Sharing|TopologyAwareScheduling)" \
+  --skip="(AppWrapper|JobSet|LeaderWorkerSet|JAX|Kuberay|Metrics|Fair Sharing|TopologyAwareScheduling|Kueue visibility server|Failed Pod can be replaced in group|should allow to schedule a group of diverse pods)" \
```

Comment:
should we also skip `should allow to change queue name if ReadyReplicas=0`?
It's skipped in release-0.11

Reply:
It hasn't failed on any tests actually so I may have fixed it?

Reply:
Sure. If you would like, you could also proposed it in upstream